Advanced Usage documentation reworked 2 (#1513)
onionshare/onionshare: docs/source/advanced.rst

Save Tabs
---------

Everything in OnionShare is temporary by default. Addresses no longer exist and can't be used again when OnionShare tabs are closed.
Your OnionShare service can also be persistent.

> If host a website, percistence means it will be available on the same OnionShare address even if you reboot your computer.

Reviewer: Instead of, "If host a website" it should be, "If you host a website"

To make any tab persistent, check the "Save this tab, and automatically open it when I open OnionShare" box before starting your server.
A purple pin icon appears to the left of its server status to tell you the tab is saved.

.. image:: _static/screenshots/advanced-save-tabs.png

When opening OnionShare, your saved tabs from the prior session will start opened.
Each service then can be started manually, and will be available on the same OnionShare address and be protected by the same private key.

> If you save a tab, a copy of its onion service secret key is stored on your computer with your OnionShare settings.

Author: "with your OnionShare settings" isn't very clear. In the settings, settings for (?) is also saved alongside the key?

Reviewer: What this means is that OnionShare settings are all stored in the same folder on the disk (in different places depending on the user's OS), and it's letting them know that secret keys will be stored in that folder as well. Though it could just say, "f you save a tab, a copy of its onion service secret key is stored on your computer."

Reviewer: "with your OnionShare settings" can just be removed.

> If you save a tab, a copy of its onion service secret key is stored on your computer with your OnionShare settings.

Author: ```suggestion
If you save a tab, a copy of its onion service secret key is stored on your computer.
```

When browsing to an OnionShare service in Tor Browser, Tor Browser will prompt for the private key to be entered.

> You can make a OnionShare service accessible to the public. Set up an OnionShare receive service so the public can securely and anonymously send you files.

Reviewer: The receive mode service is just a single example of when you might want to make a service public. Instead of, "Set up an OnionShare receive service", it should be something like, "For example, you can set up an OnionShare receive service"

> You can make a OnionShare service accessible to the public. Set up an OnionShare receive service so the public can securely and anonymously send you files.

Author: ```suggestion
You can files accessible to the public by setting up an OnionShare receive service.
```

    onionshare-cli --help

> Info about installing it on different operating systems can be found in the `CLI README file <https://github.com/onionshare/onionshare/blob/develop/cli/README.md>`_ in the Git repository.

Author: As per https://no.wikipedia.org/wiki/Git
